pkg/schedule/schedulers/evict_slow_trend.go

```diff
@@ -304,17 +294,11 @@ func (s *evictSlowTrendScheduler) EncodeConfig() ([]byte, error) {
 func (s *evictSlowTrendScheduler) ReloadConfig() error {
 	s.conf.Lock()
 	defer s.conf.Unlock()
-	cfgData, err := s.conf.storage.LoadSchedulerConfig(s.GetName())
-	if err != nil {
-		return err
-	}
-	if len(cfgData) == 0 {
-		return nil
-	}
 	newCfg := &evictSlowTrendSchedulerConfig{}
-	if err = DecodeConfig([]byte(cfgData), newCfg); err != nil {
+	if err := s.conf.load(newCfg); err != nil {
```

Comment:
How about `err, newCfg := s.conf.load()`

Reply:
We need to know the structures that need to be decoded, so we have to input them in.